src/libraw.mk

```diff
@@ -0,0 +1,31 @@
+# This file is part of MXE. See LICENSE.md for licensing information.
+
+PKG             := libraw
+$(PKG)_WEBSITE  := https://libraw.org
+$(PKG)_DESCR    := A library for reading RAW files obtained from digital photo cameras
+$(PKG)_VERSION  := 0.19.5
+$(PKG)_CHECKSUM := 9a2a40418e4fb0ab908f6d384ff6f9075f4431f8e3d79a0e44e5a6ea9e75abdc
+$(PKG)_GH_CONF  := LibRaw/LibRaw/releases
+$(PKG)_DEPS     := cc jpeg lcms
+
+define $(PKG)_UPDATE
+    echo 'TODO: write update script for $(PKG).' >&2;
+    echo $($(PKG)_VERSION)
+endef
+
+define $(PKG)_BUILD
+    cd '$(SOURCE_DIR)' && autoreconf -fi
+    cd '$(BUILD_DIR)' && '$(SOURCE_DIR)'/configure \
+        $(MXE_CONFIGURE_OPTS) \
+        --disable-jasper \
+        --enable-jpeg \
+        --enable-lcms \
+        --disable-examples \
+        CXXFLAGS='$(if $(BUILD_SHARED),-DLIBRAW_BUILDLIB,-DLIBRAW_NODLL)' \
+        LDFLAGS='-lws2_32'
+    $(MAKE) -C '$(BUILD_DIR)' -j '$(JOBS)' install
+
+    '$(TARGET)-g++' -Wall -Wextra -std=c++11 $(if $(BUILD_STATIC),-DLIBRAW_NODLL) \
```

Comment:
Add -DLIBRAW_NODLL to Cflags.private in *.pc file

Reply:
Done! I opted to append these lines from the recipe instead of using a patch file. Let me know if this is OK, or whether you prefer patching.

Reply:
Looks good.